src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -179,8 +179,7 @@ function MyServices(): React.JSX.Element {
             {
                 key: 'details',
                 label:
-                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DESTROY_SUCCESSFUL ||
-                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DEPLOYMENT_FAILED ? (
+                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DESTROY_SUCCESSFUL ? (
```

Comment:
@baixinsui Actually the condition was added incorrectly and i had requested him to open a PR to rollback this wrong fix. Its better if you revert this complete commit from him with your PR. Thank you

https://github.com/eclipse-xpanse/xpanse-ui/pull/364/files

Reply:
@baixinsui we must remove this full condition itself - whatever that was added in PR-364. It was completely wrong. 
the actual requirement was to disable details button for other conditions and this user story is still open.